airflow/providers/apache/spark/operators/spark_jdbc.py

```diff
@@ -44,14 +44,6 @@ class SparkJDBCOperator(SparkSubmitOperator):
     :param spark_files: Additional files to upload to the container running the job
     :param spark_jars: Additional jars to upload and add to the driver and
                        executor classpath
-    :param num_executors: number of executor to run. This should be set so as to manage
-                          the number of connections made with the JDBC database
-    :param executor_cores: Number of cores per executor
-    :param executor_memory: Memory per executor (e.g. 1000M, 2G)
-    :param driver_memory: Memory allocated to the driver (e.g. 1000M, 2G)
-    :param verbose: Whether to pass the verbose flag to spark-submit for debugging
-    :param keytab: Full path to the file that contains the keytab
-    :param principal: The name of the kerberos principal used for keytab
```

Comment:
Why are these removed?

Reply:
All of these params exist already in the parent, so to avoid repetition, we could treat them in the `**kwargs` passed to `super().__init__(...)`.

Reply:
cool thanks!